curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6555df0c81300b175308557d.md

```diff
@@ -1,20 +1,27 @@
 ---
 id: 6555df0c81300b175308557d
-title: Step 84
+title: Step 89
 challengeType: 20
-dashedName: step-84
+dashedName: step-89
 ---
 
 # --description--
 
-Below the line you just modified, add another `print()` call to print `Key: python`. Again, use concatenation (`+`) to get this result.
+Below the `print()` call you just added, add another `print()` call to print `Key: python`. Again, use concatenation (`+`) to get this result.
```

Comment:
Here you could remind of the `custom_key` variable in the description too, other than in the hint